plugins/hydra_rq_launcher/tests/test_rq_launcher.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import pytest  # type: ignore
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+from hydra.test_utils.launcher_common_tests import (
+    IntegrationTestSuite,
+    LauncherTestSuite,
+)
+from hydra.test_utils.test_utils import TSweepRunner, chdir_plugin_root
+
+from hydra_plugins.hydra_rq_launcher.rq_launcher import RQLauncher
+
+chdir_plugin_root()
+
+
+def test_discovery() -> None:
+    # Tests that this plugin can be discovered via the plugins subsystem when looking for Launchers
+    assert RQLauncher.__name__ in [
+        x.__name__ for x in Plugins.instance().discover(Launcher)
+    ]
+
+
+@pytest.mark.parametrize("launcher_name, overrides", [("rq", [])])
+class TestRQLauncher(LauncherTestSuite):
+    """
+    Run the Launcher test suite on this launcher.
+    """
+
+    pass
+
+
+@pytest.mark.parametrize(
+    "task_launcher_cfg, extra_flags",
+    [
+        (
+            {
+                "defaults": [
+                    {"hydra/launcher": "rq"},
+                    {"hydra/hydra_logging": "hydra_debug"},
+                    {"hydra/job_logging": "disabled"},
```

Comment:
I know this is copy paste.
can you check what happens if you don't override the logging?
does it print new shit when you run the test or is it okay?

Reply:
If I take out lines L39-40 tests pass as before, no extra output is printed

Reply:
Changed the tests to not override the logging

Reply:
let's do it.